docs/api/node.md

```diff
@@ -311,7 +311,15 @@ enum InterviewStage {
 readonly deviceClass: DeviceClass
 ```
 
-This property returns the node's [DeviceClass](#DeviceClass-class), which provides further information about the kind of device this node is.
+This property returns the node's **DeviceClass**, which provides further information about the kind of device this node is.
+
+<!-- #import DeviceClass from "zwave-js" -->
+
+<!-- #import BasicDeviceClass from "zwave-js" -->
+
+<!-- #import GenericDeviceClass from "zwave-js" -->
```

Comment:
@AlCalzone Should this be `@zwavejs/config`?

Reply:
yup

Reply:
> 
> 
> > I'll need to change the output for classes to not include the implementation.
> In another pr or this?

this